src/components/Tag/Tag.jsx

```diff
@@ -71,15 +89,14 @@ const Text = styled(Typography)`
 `
 
 Tag.defaultProps = {
-  selected: true,
-  close: false,
-  color: 'primary'
+  close: false
 }
 
 Tag.propTypes = {
-  selected: PropTypes.oneOf([true, false, 'disabled']),
+  variant: PropTypes.oneOf(['selected', 'unselected', 'disabled']),
```

Comment:
Eu tinha discutido com o edu sobre essa variant `disabled`, a gente ficou na dúvida se o disabled deveria ser outra prop, tipo: Uma tag com o estilo de `selected` porém `disabled` (cinzinha), o mesmo valeria pra `unselected`. Tens como trocar essa ideia com o pessoal do design?

Reply:
tu não acha valido então, retirar a variant, e deixar a prop `selected` que recebe true ou false, e um `disabled` q recebe true ou false. Pq talvez variant não faça sentido, ja q meio q altera a funcionalidade do componente

Reply:
ai é q ta, tipo, eu n sei se faz sentido ser selected tmb sabe. Fiquei meio na dúvida se isso era pra ser só uma variação de design da parada ou se realmente é pra quando clicar na tag ela ficar desse jeito

Reply:
Se for "selected" mesmo (quando clica fica selected) ai sim q faz sentido ter um boolean `selected` e outro `disabled`

Reply:
Tem como tu ver com o @naveteam/designers se isso é só uma variacao de style ou uma funcionalidade de selecionar/desselecionar?